menpo/base.py

```diff
@@ -1,25 +1,6 @@
-from functools import partial
+from functools import partial, wraps
 import os.path
 
-# To debug the Copyable interface, simply uncomment lines 11-23 below and the
-# four lines in the copy() method.
-# Then you can call print_copyable_log() to see exactly what types have been
-# skipped in copying and why.
-
-# from collections import defaultdict
-# alien_copies = defaultdict(set)
-# non_copies = defaultdict(set)
-#
-#
-# def print_copyable_log():
-#     print('Has .copy() but not Copyable:')
-#     for k, v in alien_copies.items():
-#         print('  {:15}|  {}'.format(k, ', '.join(v)))
-#
-#     print('\nNo .copy() (shallow copied):')
-#     for k, v in non_copies.items():
-#         print('  {:15}|  {}'.format(k, ', '.join(v)))
-
```

Comment:
Think this might be useful to leave in? Or if you feel strongly about removing it get rid of the four lines referenced in Copyable too.

No problem with you removing it, I'll just make a little gist with it for future reference.